acceptance/Gemfile

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+
+gem 'chef-acceptance', github: 'chef/chef-acceptance'
+gem 'berkshelf'
```

Comment:
My assumption is that chefdk will be installed.  Which I think is a good thing to enforce for acceptance.  We can always add them here if we run into issues.  I'll add that to the README.